db/schema.cds

```diff
@@ -12,43 +12,36 @@ namespace sap.fe.cap.travel;
 entity Travel : managed {
   key TravelUUID : UUID;
   TravelID       : Integer @readonly default 0;
-  BeginDate      : Date;
-  EndDate        : Date;
-  BookingFee     : Decimal(16, 3);
-  TotalPrice     : Decimal(16, 3) @readonly;
-  CurrencyCode   : Currency;
+  BeginDate      : Date @mandatory;
+  EndDate        : Date @mandatory;
+  BookingFee     : Decimal(16,3) @mandatory default 11;
```

Comment:
default and mandatory?

Reply:
@danjoa : what is the purpose of having both `@mandatory` and `default` for a Decimal value?

Reply:
True, makes no sense